src/languages/en-US.ts

```diff
@@ -27,6 +27,7 @@ export default class extends Language {
 This will **not** rename existing users`,
         PING_COMMAND_DESCRIPTION: "Shows you my ping to discord's servers",
         PING_INITIAL_MESSAGE: "Pinging...",
+        VOTE_COMMAND_DESCRIPTION: "Vote for Fire to get premium.",
```

Comment:
```suggestion
        VOTE_COMMAND_DESCRIPTION: "Vote for Fire to get premium.",
```
??????????

Reply:
what did i do wrong now

Reply:
> what did i do wrong now

since when does voting give you premium? I made the damn bot and I didn't know this

Reply:
oh oops, i miss interpreted it

Reply:
ok fixed